c7n/cli.py

```diff
@@ -278,14 +278,21 @@ def setup_parser():
         "Validate config files against the json schema")
     validate = subs.add_parser(
         'validate', description=validate_desc, help=validate_desc)
-    validate.set_defaults(command="c7n.commands.validate")
+    validate.set_defaults(command="c7n.commands.validate", check_deprecations="yes")
     validate.add_argument(
         "-c", "--config", help=argparse.SUPPRESS)
     validate.add_argument("configs", nargs='*',
                           help="Policy Configuration File(s)")
     validate.add_argument("-v", "--verbose", action="count", help="Verbose Logging")
     validate.add_argument("-q", "--quiet", action="count", help="Less logging (repeatable)")
     validate.add_argument("--debug", default=False, help=argparse.SUPPRESS)
+    deprecations = validate.add_mutually_exclusive_group(required=False)
+    deprecations.add_argument("--skip-deprecations", dest="check_deprecations",
+                              action='store_const', const='skip',
+                              help="Do not check for deprecations")
+    deprecations.add_argument("--deprecations-as-errors", dest="check_deprecations",
```

Comment:
wondering about the wordiness of spelling out deprecations here affecting cli ux, vs a short form --dep-error, and --no-deps

Reply:
Thinking overnight, I thought perhaps --strict for treating deprecations as errors.
Happy to have --no-deps for skipping deprecations.